working/augmentation-libraries/feature-specification.md

```diff
@@ -66,51 +66,51 @@ part of macro expansion, it's still useful to have it be in a canonical
+of the changes that the macros made to the library where they are applied. The
+language then automatically merges those library augmentations into the
+augmented library.
 
-But augmentations aren't *only* a serialization format for macros. They are a
-first-class language feature that can be produced by non-macro code generators
-or written by hand by users who simply want to break a giant library or class
-into smaller files.
+But library augmentations aren't *only* a serialization format for macros. They
+are a first-class language feature that can be produced by non-macro code
+generators or written by hand by users who simply want to break a giant library
+or class into smaller files.
 
-## Augmentations
+## Library Augmentations
 
-An augmentation is a separate file that *augments* another *main* library. An
-augmentation is somewhere between a separate library and a part file.
+A library augmentation is a separate file that *augments* an existing library,
+referred to as the *augmented* library. A library augmentation is similar to
+but more powerful than a part file, and it is not a library in its own right.
 
 *   Like a library, it may contain all kinds of declarations&mdash;functions,
     classes, variables, etc.
 
 *   Like a library, it has its own import scope and may contain its own imports.
 
 *   Like a part file, all of the top-level declarations it produces end up in
-    the top-level scope of the main library.
+    the top-level scope of the augmented library.
 
-*   Like a part file, it shares a private scope with the main library and the
-    two have mutual access to private declarations in the other file.
+*   Like a part file, it shares a private scope with the augmented library and
+    the two have mutual access to private declarations in the other file.
 
-Augmentations have a few features unique to them:
+Library augmentations may also contain declaration augmentations, which augment
+existing declarations from the library. Some examples include:
 
-*   An augmentation may add new members to existing types in the main library,
-    including adding new values to enums.
+*   Type augmentations, which can add new members to types, including adding new
+    values to enums, or even alter the type hierarchy by adding mixins, etc.
 
-*   A function in the augmentation may wrap the body of a function in the main
-    library, or provide a body if none was present.
+*   Function augmentations, which can replace the body of a function, or provide
+    a body if none was present.
 
-*   A variable in the augmentation may wrap the initializer of a variable in the
-    main library, or provide an initializer if none was present.
-
-*   An enum value in the augmentation may replace the argument list of an enum
-    value in the main library, or provide an argument list if none was present.
+*   Variable augmentations may wrap the initializer of a variable in the
```

Comment:
Because I'm a tech writing nerd now and list bullets lists to have parallel grammar structure:

> Variable augmentations, which can wrap...